### PR TITLE
Feat/incomplete routes filter

### PR DESCRIPTION
**Pull request by Chi-Bottle-Nguyen**
Adding filters for incomplete routes and happening routes.
filters

### CODE REVIEW

File: src/components/Routes/Routes.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function Routes({ initial_filter }) {
   const users = useUserData()
   const [routes, setRoutes] = useState()
   const [loading, setLoading] = useState(true)
-  const [filter, setFilter] = useState(true)
+  const [filter, setFilter] = useState('all')
```
Comment (nhatduy227) [inline src/components/Routes/Routes.js:26]: suggest to change to: useState(admin ? all : mine)

**Review by nhatduy227 — CHANGES_REQUESTED** [review]
LGTM! Just wanna add a minor change so that driver will see their route first

File: src/components/Routes/Routes.js (hunk 1 above)
Comment (Chi-Bottle-Nguyen) [reply src/components/Routes/Routes.js:26]: I'll implement the changes

**Review by kennyallex2 — APPROVED** [review]
A suggestion I have is to change "Happening" to something like "Ongoing" or "Current". Besides that and Nom's fix, everything else looks great.

**Review by nhatduy227 — APPROVED** [review]
LGTM!

**Pull request merged**